modules/wpml/wpml-config.php

```diff
@@ -262,6 +279,83 @@ public function translate_taxonomies( $taxonomies, $hide ) {
 		return $taxonomies;
 	}
 
+	/**
+	 * Translation management for strings in blocks content.
+	 *
+	 * @since 3.3
+	 *
+	 * @param array[] $parsing_rules Rules as Xpath expressions to evaluate in the blocks content.
+	 * @return array[] Rules completed with ones from wpml-config file.
+	 *
+	 * @phpstan-param array<string,array<string>> $parsing_rules
+	 * @phpstan-return array<string,array<string>> $parsing_rules
+	 */
+	public function translate_blocks( $parsing_rules ) {
+		return $this->extract_blocks_parsing_rules( $parsing_rules, 'xpath' );
+	}
+
+	/**
+	 * Translation management for block attributes.
+	 *
+	 * @since 3.3
+	 *
+	 * @param array[] $parsing_rules Rules for block attributes to translate.
+	 * @return array[] Rules completed with ones from wpml-config file.
+	 *
+	 * @phpstan-param array<string,array<string>> $parsing_rules
+	 * @phpstan-return array<string,array<string>>
+	 */
+	public function translate_blocks_attributes( $parsing_rules ) {
+		return $this->extract_blocks_parsing_rules( $parsing_rules, 'key', true );
+	}
+
+	/**
+	 * Extract what kind of string to translate for blocks from WPML config file.
+	 *
+	 * @since 3.3
+	 *
+	 * @param array[] $parsing_rules         Rules to complete with ones from wpml-config file..
+	 * @param string  $child_tag             Tag name to extract.
+	 * @param bool    $is_in_child_attribute Extract tag value in attribute or not. Default false.
+	 * @param string  $child_attribute_name  Attribute name where to extract the value. Default 'name'. Used if $is_in_child_attribute is set to true.
+	 * @return array[] Rules completed with ones from wpml-config file.
+	 */
+	protected function extract_blocks_parsing_rules( $parsing_rules, $child_tag, $is_in_child_attribute = false, $child_attribute_name = 'name' ) {
+		foreach ( $this->xmls as $xml ) {
+			$blocks = $xml->xpath( 'gutenberg-blocks/gutenberg-block' );
+			if ( ! is_array( $blocks ) ) {
+				continue;
+			}
+			foreach ( $blocks as $block ) {
+				$attributes = $block->attributes();
+				if ( empty( $attributes ) ) {
+					continue;
+				}
+				if ( '1' === (string) $attributes['translate'] ) {
```

Comment:
Same here for intendation.

Reply:
See https://github.com/polylang/polylang/pull/1127/files#r985464707